verification/curator-service/ui/src/components/AutomatedBackfill.tsx

```diff
@@ -0,0 +1,233 @@
+import * as Yup from 'yup';
+
+import { Button, Typography } from '@material-ui/core';
+import { Form, Formik } from 'formik';
+import Source, { CaseReferenceForm } from './common-form-fields/Source';
+
+import AppModal from './AppModal';
+import { DateField } from './common-form-fields/FormikFields';
+import MuiAlert from '@material-ui/lab/Alert';
+import { Paper } from '@material-ui/core';
+import React from 'react';
+import axios from 'axios';
+import { makeStyles } from '@material-ui/core/styles';
+
+interface RetrievalResult {
+    bucket: string;
+    key: string;
+    upload_id: string;
+}
+
+interface User {
```

Comment:
please use the user interface that's defined in User.tsx

Reply:
Neat, thanks.